.github/workflows/yamllint.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name: yamllint
+on:
+  pull_request:
+    types: [opened, edited, synchronize, reopened]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install yamllint
+        run: pip install yamllint
```

Comment:
Should we use an image which has yamllint already installed? (e.g. https://hub.docker.com/r/cytopia/yamllint). That way we won't have to install it every time (once the image has been pulled into Prow).

Reply:
My initial plan was the same and went this way, because I could not find an official image. But perhaps it is fine to use the one referenced.

Reply:
Just heads up, that this is running on GitHub infra.  @macaptain  Do you think it is ok to use https://hub.docker.com/r/cytopia/yamllint ?

Reply:
This looks interesting, we can leave it as is as I found [this](https://github.com/lsst/pipe_base/blob/main/.github/workflows/yamllint.yaml#L18-L19) where it is installed the same way or maybe we could use this [one](https://github.com/frenck/action-yamllint#usage)?

Reply:
nice, I think using the GA image is the best, I will test it

Reply:
I found `ibiqlik/action-yamllint@v3` image to be more stable and reliable.